modules/4337/test/erc4337/ERC4337ModuleExisting.spec.ts

```diff
@@ -54,22 +59,28 @@ describe('Safe4337Module - Existing Safe', () => {
         '0x',
         '0',
         await entryPoint.getAddress(),
+        false,
+        false,
+        {
+          maxFeePerGas: '0',
+        },
```

Comment:
Interesting... I would have emulated this by doing a:

```
await entryPoint.depositTo(safe.address, { value: ethers.parseEther("1.0") })
```

Reply:
Hmm, I might've understood `without fee` too literally...